cli-wrap/src/main/resources/cli/da-cli.py

```diff
@@ -339,6 +339,15 @@ def list_black(self):
         else:
             print("Bad arguments! For help use: ./da-cli.py --help")
             
+    def list_white(self):
+        if len(sys.argv) == 3:
+            self.da.print_white_artifacts()
+        elif len(sys.argv) == 4:
+             if self.pm.matchProd(sys.argv[3]):
+                self.da.print_white_artifacts(sys.argv[3])
+        else:
+            print("Bad arguments! For help use: ./da-cli.py --help")
+            
```

Comment:
But this doesn't have to do anything with WS, right? So at least put it into separate commit if not PR, please.